clients/src/main/java/org/apache/kafka/common/internals/IdempotentCloser.java

```diff
@@ -0,0 +1,55 @@
+/*
```

Comment:
let's move this to the refactor PR and use it in the subsequent CTR PR.

Reply:
but awesome util!

Reply:
Done.